systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java

```diff
@@ -62,4 +62,11 @@ public static long getTimeoutForKafkaRebalanceState(KafkaRebalanceState state) {
 
         return timeout;
     }
+
+    /**
+     * rollingUpdateTimeout returns a reasonable timeout in milliseconds for a number of pods in a to roll on update
```

Comment:
Sorry I left a typo in my recommended update:

for a number of pods in a **quorum** to roll on update

Reply:
Yeah sorry, I just copy paste it :D